src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1616,9 +1616,12 @@ QueueToadlet.insertFileBrowseLabel=Upload a file from disk
 QueueToadlet.insertFileBrowseButton=Browse files
 QueueToadlet.insertFileResetForm=Reset Form
 QueueToadlet.key=Key
-QueueToadlet.lastActivity=Last Activity
+QueueToadlet.lastActivity=Last Progress
```

Comment:
Okay, that makes sense. Comments are possible in some l10n formats but not this one.